### DISCUSSION TITLE
"Hidden" Changes in version 6.8.0

### DISCUSSION THREAD

**Discussion by uporersith** [firemodels/fds#11969] [General]
Not a big deal, but I noticed in version 6.8.0, &MISC SURF_DEFAULT='MY SURF' now throws an error "PROBLEM WITH MISC line" and that the FED output quantity: "INCAPACITATION TIME" has been removed. I tried searching the issues, discussions, commit log messages for some hints as to why just out of curiosity as I am interested in the reasons why things are put in/taken out. 

Just wondering if there was any more juicy details behind these changes. thanks!!

**Comment by mcgratta** [comment]
Not really. Over the years, the `MISC` group of parameters has grown too big, and we prefer to set `DEFAULT=T` directly on the appropriate `SURF` line. We maintained backward compatibility for a few years, and then dropped it. Hopefully we removed it from the User's Guide. If not let me know. As for `INCAPACITATION TIME`, I'll let someone else answer. Not sure about that one.
